.github/workflows/gradle-release.yml

```diff
@@ -0,0 +1,32 @@
+name: Gradle Release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-java@v4
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+        cache: gradle
+
+    - name: Grant execute permission to Gradle wrapper
+      run: chmod +x ./gradlew
+
```

Comment:
@tuancoltech Since you added the executable flag to the `gradlew` file, this code is probably no longer needed.

```suggestion
```